test/performance/odp_ipsec.c

```diff
@@ -277,6 +277,10 @@ static ipsec_alg_config_t algs_config[] = {
 				.data = test_key16,
 				.length = sizeof(test_key16)
 			},
+			.cipher_key_extra = {
+				.data = test_salt,
+				.length = 4,
+			},
```

Comment:
Could you please separate these two hunks into separate commit: I'd like to get them into master branch as well.

Reply:
Done in v2